setup.py

```diff
@@ -31,7 +31,7 @@ def read(fname):
 
 setup(
     name="pytest-inmanta-lsm",
-    version="1.11.0",
```

Comment:
I don't think you want to commit that change here, this should only be done on the next branch.

Reply:
I am struggling a bit with the release process for this package.  What we merge as a patch should not get a `1.11.0.dev...` version but rather a `1.10.1.dev...` (while waiting for the actual release), which is why I made this change, and I was indeed planning on changing it once more in the next branch to bring it back to `1.10.0` so that this bumpversion action would bring it to the correct version: https://github.com/inmanta/irt/blob/master/ci/inmanta-tools/Jenkinsfile-pyp#L201

Is there a better way of doing this?